waspc/data/Generator/templates/server/src/routes/auth/passport/github/github.js

```diff
@@ -0,0 +1,19 @@
+import { configFn } from './config.js'
+export { getUserFieldsFn } from './config.js'
+
+// Validates the provided config function returns all required data.
+export const config = ((config) => {
+  if (!config?.clientID) {
+    throw new Error("auth.gitHub.configFn must return an object with a clientID property.")
```

Comment:
Ha no worries, yeah I actually had to renormalize the spelling a few times for it, but figured if we are camel casing from lowercase that `gitHub` was probably best.

Hmm, I think in this case it may help the user though, by telling them where it is missing. But yeah it is an easily forgettable thing to update should this change.

Maybe something more generic like: `The GitHub configFn must return ....`? I think that prob works too! 👍🏻

Reply:
Updated wording 👍🏻